### DISCUSSION TITLE
Creating two FastCGI apis in one program

### DISCUSSION THREAD

**Discussion by ThenoobMario** [AxisCommunications/acap-native-sdk-examples#236] [Q&A] [answered]
Hi,

I want to create two FastCGI APIs that perform separate tasks in my program. I followed the `web-server-using-fastcgi` example and created one cgi with it. How do I create a second cgi? What would the code structure look like?

**Comment by pataxis** [comment, marked as answer]
You can specify a second CGI end-point in the manifest.json.

```json
        "configuration": {
            "httpConfig": [
                {
                    "access": "admin",
                    "name": "first.cgi",
                    "type": "fastCgi"
                },
                {
                    "access": "viewer",
                    "name": "second.cgi",
                    "type": "fastCgi"
                },
            ]
        }
```

Then you can redirect calls from different end-points to different parts of your program.
